tests/fixtures/init/golden/pkg-desc/pkg-with-comments.golden

```diff
@@ -0,0 +1,46 @@
+cabal-version:      2.4
+
+-- Initial package description 'y' generated by
+-- 'cabal init'. For further documentation, see:
+--   http://haskell.org/cabal/users-guide/
+-- 
+-- The name of the package.
+name:               y
+
+-- The package version.
+-- See the Haskell package versioning policy (PVP) for standards
+-- guiding when and how versions should be incremented.
+-- https://pvp.haskell.org
+-- PVP summary:     +-+------- breaking API changes
+--                  | | +----- non-breaking API additions
+--                  | | | +--- code changes with no API change
+version:            0.1.0.0
+
+-- A short (one-line) description of the package.
+synopsis:           synopsis
+
+-- A longer description of the package.
+-- description:
+
+-- URL for the project homepage or repository.
+homepage:           home
+
+-- The license under which the package is released.
+license:            BSD-3-Clause
+
+-- The file containing the license text.
+license-file:       LICENSE
+
+-- The package author(s).
+author:             foo-kmett
+
+-- An email address to which users can send suggestions, bug reports, and patches.
+maintainer:         foo-kmett@kmett.kmett
+
+-- A copyright notice.
+-- copyright:
+category:           Data
+build-type:         Simple
+
+-- Extra files to be distributed with the package, such as examples or a README.
+extra-source-files: CHANGELOG.md
```

Comment:
The PR supposedly fixes #6864, but this doesn't list the changelog as `extra-doc-files`. How so?

Reply:
We're finding a few bugs in a few places like that as we go along with our internal review. It seems we marked "ready to review" a little prematurely. I'll ping when we have this fixed (it's simple fix)

Reply:
I'm still seeing this.

Reply:
Where are you still seeing this? It doesn't match the golden files I'm seeing: https://github.com/haskell/cabal/blob/76468fbedbbc92e613fceaf43057de24caeebc1a/cabal-install/tests/fixtures/init/golden/pkg-desc/pkg-with-comments.golden

Reply:
I think github was lying to me. I see the problem no longer. :)

Reply:
I actually found the problem (and it was a problem!): check out the file name. It's at the root of the project! Golden files must have been run from the root using `cabal run cabal-install:unit-tests`, which creates the golden files at whatever the current directory is :angry:

I've cleared out the stale golden files.